lib/reporters/html/reporterTemplate.html

```diff
@@ -0,0 +1,204 @@
+                            {{#with failed}}
+                                <td>{{iterations}}</td>
+                                <td>{{requests}}</td>
+                                <td>{{testScripts}}</td>
+                                <td>{{preRequestScripts}}</td>
+                                <td>{{assertions}}</td>
+                            {{/with}}
+                        </tr>
+                    </tbody>
+
+                    <tfoot></tfoot>
+                </table>
+            {{/with}}
+
+            <br>
+
+            {{#with statistics}}
+                <table>
+                    <thead>
+                        <tr>
+                            <th colspan="3">
+                                <h1>Run Statistics</h1>
+                            </th>
+                        </tr>
+                        <tr>
+                            <th>Total run duration</th>
+                            <th>Bytes transferred out</th>
+                            <th>Bytes transferred in</th>
+                        </tr>
+                    </thead>
+
+                    <tbody>
+                        <tr>
+                            <td>{{runDuration}}</td>
+                            <td>{{bytes.out}}</td>
+                            <td>{{bytes.in}}</td>
+                        </tr>
+                    </tbody>
+                </table>
+            {{/statistics}}
+
+            <br>
+
+            {{#with metadata}}
+                <table>
+                    <thead>
+                        <tr>
+                            <th colspan="6">
+                                <h1>Run Metadata</h1>
+                            </th>
+                        </tr>
+
+                        <tr>
+                            <th colspan="2">Environment variables</th>
+                            <th colspan="2">Global variables</th>
+                            <th colspan="2">Data variables</th>
+                        </tr>
+
+                        <tr>
+                            <th>Key</th>
+                            <th>Value</th>
+                            <th>Key</th>
+                            <th>Value</th>
+                            <th>Key</th>
+                            <th>Value</th>
+                        </tr>
+                    </thead>
+
+                    <tbody>
+                        <tr>
+                            <td>{{Value}}</td>
+                            <td>{{Value}}</td>
+                            <td>{{Value}}</td>
+                            <td>{{Value}}</td>
+                            <td>{{Value}}</td>
+                            <td>{{Value}}</td>
+                        </tr>
+                    </tbody>
+
+                    <tfoot></tfoot>
+                </table>
+            {{/metadata}}
+
+            <br>
+
+            {{#each folders}}
+                {{#with this}}
+                    <table>
```

Comment:
Every item in a folder is effectively an object, hence using the context wrapper `{{#with this}}` saves the effort of having to type `this.property` for every instance. `{{#each ...}}` would simply iterate over the elements of the specified array.